V2/time_tracker/time_entries/_domain/_use_cases/_get_time_entries_summary_use_case.py

```diff
@@ -0,0 +1,11 @@
+from time_tracker.time_entries._domain import TimeEntry, TimeEntryService
+import typing
```

Comment:
Following the order of import that was defined, typing would be first then a blank line then the another import